### ISSUE TITLE
Poor contrast ratio on red circles with black text

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/hourglass#243]
## Report Issue

**To Reproduce**
Steps to reproduce:
1. Go to ['open access movement page'](https://olhbeta.janeway.systems/site/open-access-movement/)
2. Scroll down to 1997.  Then every alternate entry is themed in red, with black text on a red circle for the year heading.
3. Contrast ratio of black on red is 4.24:1 which is a fail.

**Screenshots**
<img width="255" alt="image" src="https://github.com/BirkbeckCTP/hourglass/assets/5330770/1129dcc0-46b8-4d2e-b524-44440bbd5ba9">

## Preferred Solution
Instead of solid red and orange circles, have these as rings, with the background colour as an inner circle, such that the text is the same as the accompanying paragraph - nb this means updating the orange circles as well as the red ones.

e.g.
<img width="268" alt="image" src="https://github.com/BirkbeckCTP/hourglass/assets/5330770/3a3787cc-0ab2-4001-b8e0-3466f845ecd7">

**Comment by StephDriver** [comment]
As an alternative to the solution above, how about this:

The black we use on the site (`#202124), is not the darkest black there is, but neighbouring colours do affect the colour we see - so I think is is possible that if we change this black to `#000000` just for the text within red circles, we'd then have appropriate contrast, and user's won't even notice it wasn't the same black as the paragraph nearby that is against a tan colour.  Here's the example:



![Image](https://github.com/user-attachments/assets/26d0d26b-bd71-4681-9bf8-dbfe5f5aa1c2)
